File: src/main/java/com/dingdongdeng/coinautotrading/domain/repository/ExchangeCandleRepository.java
package com.dingdongdeng.coinautotrading.domain.repository;

import com.dingdongdeng.coinautotrading.domain.entity.ExchangeCandle;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ExchangeCandleRepository extends JpaRepository<ExchangeCandle, Long> {

}

File: app-service/src/main/kotlin/com/dingdongdeng/autotrading/domain/exchange/repository/ExchangeCandleRepository.kt
package com.dingdongdeng.autotrading.domain.exchange.repository

import com.dingdongdeng.autotrading.domain.exchange.entity.ExchangeCandle
import org.springframework.data.jpa.repository.JpaRepository

interface ExchangeCandleRepository : JpaRepository<ExchangeCandle, Long> {

}